packages/react/src/lib/felaDisableAnimationsPlugin.ts

```diff
@@ -0,0 +1,26 @@
+import * as _ from 'lodash'
+
+export default () => {
+  const disableAnimations = (styles: Object, type?, renderer?, props?) => {
```

Comment:
note, infinite recursion is possible in case if object has links to self in the props tree. At the very least, lets mention this limitation in the comments - it shouldn't be a problem of correct styles object, but given that this object is something that _client provides us with, it may be an accidental mistake in styles that would result in whole app hanging

Reply:
Added, please take a look. We should add this for the other plugins that potentially can cause infinite recursion.